### DISCUSSION TITLE
Home Assistant Supervised - problema di dipendenze durante installazione + jq non installato

### DISCUSSION THREAD

**Discussion by Emanuele-DV** [indomus/forum#3221] [HUB: Home Assistant]
Salve 
sto cercando di installare Home Assistant Supervised

Su Pc Intel con Debian 11 ... sono arrivatoa a questo comando che mi da errore 

wget https://github.com/home-assistant/supervised-installer/releases/latest/download/homeassistant-supervised.deb
sudo dpkg -i homeassistant-supervised.deb

problema di dipendenze durante installazione + jq non installato 

come posso risolvere ? 

Grazie Emanuele

**Comment by snakuzzo** [comment]
Ciao.
Premesso che sconsigliamo vivamente questo tipo di installazione, proprio per tutta una serie di vincoli legati ai pacchetti lato sistema operativo (non solo per il caso banale di `jq`).
Comunque, per installare `jq`...

```
sudo apt update
sudo apt install jq
```

In aggiunta, visto che chiedi info del genere per questo tipo di installazione, già posso dirti che questa installazione non fa al caso tuo. Se vai sulla [pagina ufficiale dello script di installazione](https://github.com/home-assistant/supervised-installer) infatti, trovi scritto questo...

![image](https://github.com/user-attachments/assets/1f2b7183-737f-4abd-9688-5f514d67728a)

E un advanced user non può bloccarsi all'installazione di `jq`.

Se proprio non vuoi solo Core, ma vuoi tirare su HA OS su una VM ed avere l'OS sotto a disposizione per altro, crea una VM
